wiki/Article_Style_Guide/en.md

```diff
@@ -255,7 +299,7 @@ You may use the old game mode names (e.g. `Catch the Beat` or `Taiko`) only when
 
 ### Game Modifiers
 
-Game modifiers **must** be capitalised.
+Game modifiers **must** capitalise the first letter of each word.
```

Comment:
whoops :s will change... but title casing is quite ambiguous because some prefer to caps every first letter of each word while some just caps "big" words.